src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -358,19 +438,16 @@ const useGroupInfo = (channelId = null) => {
    *
    * @param {('view' | 'remove' | 'message' | 'block' | 'message-anonymously')} status
    */
-  const alertRemoveUser = async (status) => {
+  const handleOpenPopup = async (status) => {
+    console.warn('selectedUser', JSON.stringify(selectedUser));
     if (status === 'view') {
       setOpenModal(false);
       handleOpenProfile(selectedUser).catch((e) => console.log(e));
     }
     if (status === 'remove') {
       Alert.alert(
         null,
-        `Are you sure you want to remove ${
-          selectedUser.user.name || selectedUser?.user?.username || selectedUser.username
-        } from this group? We will let the group know that you removed ${
-          selectedUser.user.name || selectedUser?.user?.username || selectedUser.username
-        }.`,
+        `Are you sure you want to remove ${selectedUser?.username} from this group? We will let the group know that you removed ${selectedUser?.username}.`,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `alertRemoveUser` function has been renamed to `handleOpenPopup` for clarity. The logic remains the same.
```diff
- const alertRemoveUser = async (status) => {
+ const handleOpenPopup = async (status) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->